internal/internal.go

```diff
@@ -224,12 +230,16 @@ func (p *envoyExtAuthzGrpcServer) eval(ctx context.Context, input ast.Value, opt
 
 		rs, err := rego.New(opts...).Eval(ctx)
```

Comment:
Reading the OPA [source](https://github.com/open-policy-agent/opa/blob/master/rego/rego.go) a bit... what happens if you add `rego.Trace(true)` to the opts line above. and then you change this line to:

```suggestion
		r := rego.New(opts...)
		rs, err := r.Eval(ctx)
		rego.PrintTrace(os.Stdout, r)  // or replace os.Stdout with whatever `io.Writer` you want
```

Reply:
This didn't seem to add any additional logging, as far as I could tell. I think that `Trace` here is referring to the AST debug-level rego tracing, which is not what we want anyway.